### PR TITLE
Support for $...BODY to match over list of stmts

### PR DESCRIPTION
**Pull request by aryx**
PR checklist:
- [x] changelog is up to date

### CODE REVIEW

**Comment by aryx** [comment]
This closes https://github.com/returntocorp/semgrep/issues/3170

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 - Scala alpha support
 - Metrics collection of project_hash in cases where git is not available
+- $...MVARS can now match a list of statements (not just a list of arguments) (#3170)
```
Comment (IagoAbal) [inline CHANGELOG.md]: `$...MVAR` in singular? It made me doubt whether you could specify more than one metavariable there, but if it seems not unless I missed something.

**Review by IagoAbal — APPROVED** [review]
Nice feature!

To make the regexes more readable we could use OCaml's raw string notation `{|...|}`.

**Comment by aryx** [comment]
Good idea; I never used that feature (it might not parse correctly though with the ocaml parser in pfff/semgrep ... :(

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 - Scala alpha support
 - Metrics collection of project_hash in cases where git is not available
+- $...MVAR can now match a list of statements (not just a list of arguments) (#3170)
```
Comment (brendongo) [inline CHANGELOG.md:11]: Will need to move changelog entry since we cut a release

**Pull request merged**